config/clients/java/template/Java/README.mustache

```diff
@@ -0,0 +1,246 @@
+# {{artifactId}}
```

Comment:
This will need to be replaced with our templated README

Reply:
Removed in [6f9804d](https://github.com/openfga/sdk-generator/pull/158/commits/6f9804d21724edeff04096b82f2a999ab7a75144)